<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaPesquisa</class>
 <widget class="QMainWindow" name="TelaPesquisa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pesquisa</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>../Imagens/icone.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);

background-color: rgb(0, 28, 72);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="Pesquisa">
   <widget class="QPushButton" name="PlanodeFundo">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-4</y>
      <width>871</width>
      <height>519</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>NoAntialias</stylestrategy>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/grr segunda tela.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>850</width>
      <height>600</height>
     </size>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Pesquisar">
    <property name="geometry">
     <rect>
      <x>644</x>
      <y>34</y>
      <width>183</width>
      <height>55</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/btn_Pesquisar.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>145</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Verificar">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>332</y>
      <width>183</width>
      <height>55</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/btn_Verificar.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>145</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="verificar">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>412</y>
      <width>251</width>
      <height>29</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;COCOGOOSE &quot;;</string>
    </property>
    <property name="text">
     <string>NOTIFICAR RESTANDO QUANTOS DIAS?</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="box_Notificar">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>440</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Bahnschrift&quot;;</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>15</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OK">
    <property name="geometry">
     <rect>
      <x>444</x>
      <y>440</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>NoAntialias</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Bahnschrift&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Apagar">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>332</y>
      <width>183</width>
      <height>55</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/btn_Apagar.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>145</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="semProduto">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>178</y>
      <width>455</width>
      <height>51</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 0, 0);
font: 22pt &quot;Bahnschrift&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>98</y>
      <width>775</width>
      <height>197</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;
color: rgb(0, 22, 57);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="headerCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>210</number>
    </attribute>
    <attribute name="headerHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="headerMinimumSectionSize">
     <number>50</number>
    </attribute>
    <column>
     <property name="text">
      <string>Produto</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Lote</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Validade</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_Voltar_2">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>8</y>
      <width>53</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/btn_Voltar.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Editar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>332</y>
      <width>183</width>
      <height>55</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/btn_Editar.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>145</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="BarraPesquisa">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>48</y>
      <width>559</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <zorder>PlanodeFundo</zorder>
   <zorder>btn_Pesquisar</zorder>
   <zorder>btn_Verificar</zorder>
   <zorder>verificar</zorder>
   <zorder>box_Notificar</zorder>
   <zorder>btn_OK</zorder>
   <zorder>btn_Apagar</zorder>
   <zorder>treeWidget</zorder>
   <zorder>semProduto</zorder>
   <zorder>btn_Voltar_2</zorder>
   <zorder>btn_Editar</zorder>
   <zorder>BarraPesquisa</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
